src/reconcile/rec_visibility.c

```diff
@@ -107,6 +110,7 @@ __rec_append_orig_value(
      * because we have to do a search for the prepared updates, which can not proceed until eviction
      * finishes.
      */
+    // WT-9417 IGNORE
     WT_ASSERT(session, oldest_upd != NULL);
```

Comment:
This is also related to column store. Can we change to a better tag?

Reply:
Out of curiosity how do we know this is related to column store as the comment above doesn't reference it?

Reply:
Oh, that's a typo. I mean not related to. I misunderstood the tag at the time.

Reply:
I think an assertion with such a large comment deserve an ASSERT_STRONG. I know that isn't a great metric but this likely was added to catch some very rare edge case. @ajmorton

Reply:
@quchenhao added this assertion a while back to fix a coverity bug which indicated a race condition, I could now go either way here.

Reply:
Nah, makes sense. Based on [Alex's comment above](https://github.com/wiredtiger/wiredtiger/pull/8004#discussion_r894052035) we're moving towards enabling a majority of asserts in release so no harm in adding this one too.
Updated in [e63995d](https://github.com/wiredtiger/wiredtiger/pull/8004/commits/e63995d83a0efd046f3ff84ad087e3441bfe6a78)